models/settings.model.js

```diff
@@ -4,15 +4,15 @@ const mongoose = require("mongoose");
 const settings = new mongoose.Schema({
     openTime: {
         type: Date,
-        default: 0
+        default: Date.now() + 2628000000 // One month from now.
```

Comment:
Maybe it'd be a good idea to have it default to a day before Date.now()?
Only time I can think of were we should be using defaults is for local environment and applications open is probably most useful state for that?

Reply:
I set it 1mo from now for production reasons. I wouldn't want the API to be able to accept applications right away.

Reply:
The other time we use defaults is when you create a new DB for production (which may or may not happen, but it could).

Reply:
Yeah, that's fair